broker-cli/proto/broker.proto

```diff
@@ -931,6 +931,25 @@ message ReleaseChannelsRequest {
   bool force = 2;
 }
 
+/**
+ * Container for channel information for a specific currency on a given market
+ */
+message ReleasedCurrency {
+  // Currency of the information
+  Symbol symbol = 1;
```

Comment:
I thought we were moving away from using the enum for `symbol` and using strings instead, right?

Reply:
@treygriffith removed it with the last commit